src/components/App/App.js

```diff
@@ -25,9 +23,7 @@ function App() {
   return (
     <Router basename={baseAppURL}>
       <ErrorBoundary>
-        <QueryParamProvider adapter={ReactRouter6Adapter}>
```

Comment:
The test changes are just to remove the `QueryParamProvider` that are no longer needed.